### DISCUSSION TITLE
Hades - Hack & Slash Game

### DISCUSSION THREAD

**Discussion by phumzxter** [elrumo/macOS_Big_Sur_icons_replacements#1205] [Icon request]
Please make a one for this, while at it give it a try. One of the best

**Comment by OMGzilla-git** [comment]
Done. I submitted a couple different ones. They should get approved sometime tomorrow.

**Reply by phumzxter** [reply]
It doesnt seem like its not up in the website yet, any idea why?

**Reply by OMGzilla-git** [reply]
> It doesnt seem like its not up in the website yet, any idea why?

Not the slightest. It's still in my "waiting" queue. Maybe the site maintainer took a day off?

**Reply by OMGzilla-git** [reply]
> > It doesnt seem like its not up in the website yet, any idea why?
> 
> Not the slightest. It's still in my "waiting" queue. Maybe the site maintainer took a day off?

They are live now. I'm thinking of making a couple more variants without the text but otherwise same background if you think that would be preferable to either of those.
